content/docs/pipelines/troubleshooting.md

```diff
@@ -0,0 +1,86 @@
++++
+title = "Troubleshooting"
+description = "Finding and fixing problems in your Kubeflow Pipelines deployment"
+weight = 65
++++
+
+This page presents some hints for troubleshooting specific problems that you
+may encounter.
+
+## Diagnosing problems in your Kubeflow Pipelines environment
+
+For help diagnosing environment issues that affect Kubeflow Pipelines, run
+the [`kfp diagnose_me` command-line tool](/docs/pipelines/sdk/sdk-overview/#kfp-cli-tool).
+
+The `kfp diagnose_me` CLI reports on the configuration of your local
+development environment, Kubernetes cluster, or Google Cloud environment.
+Use this command to help resolve issues like:
+
+*  Python library dependencies
+*  Trouble accessing resources or APIs using Kubernetes secrets
+*  Trouble accessing Persistent Volume Claims
+
+To use the `kfp diagnose_me` CLI:
+
+1.  Install the [Kubeflow Pipelines SDK](/docs/pipelines/sdk/install-sdk/).
+1.  Follow the [guide to configuring access to Kubernetes clusters][kubeconfig],
+    to update your kubeconfig with appropriate credentials and endpoint
+    information to access your Kubeflow cluster.
+    If your Kubeflow Pipelines cluster is hosted on a cloud provider like
+    Google Cloud, use your cloud provider's instructions for configuring
+    access to your Kubernetes cluster. 
+1.  Run the following command `kfp diagnose_me`.
```

Comment:
Add a colon after the word command? Or else just "Run the `kfp diagnose_me` command."